model/Image.php

```diff
@@ -123,25 +123,13 @@ public function getCMSFields() {
 	}
 
 	/**
-	 * An image exists if it has a filename.
-	 * Does not do any filesystem checks.
```

Comment:
Well, it looks like this function is unchanged since being imported for the first time into open source.

https://github.com/dhensby/silverstripe-framework/commit/4a5d9b03f82d383cef40602aed7353dea17c31b4

The documentation about this strange behaviour seems to have been added since then, rather than at the same time.